change/@microsoft-teams-js-e4a3d833-89f1-40a8-a811-80009371d42a.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "gitmagic additions to 2.0-preview for March 29, 2022. Changes merged in from V1 include: enabling the following APIs in meeting stage: shareAppContentToStage, getAppContentStageSharingCapabilities, stopSharingAppContentToStage, and getAppContentStageSharingState",
```

Comment:
What does it mean by enabling the APIs? I do see that they are needing more FrameContexts to be initialized, but I'm still not sure.